app/controllers/api/json/imports_controller.rb

```diff
@@ -83,6 +83,42 @@ def create
     end
   end
 
+  # -------- Cancel a running import (using update PUT request) -------
+
+  def update
```

Comment:
Making this respond to `DELETE` requests might be a bit more appropriate for what the endpoint ends up doing thought I don't feel especially strongly about it.